homeassistant/components/bosch_alarm/binary_sensor.py

```diff
@@ -0,0 +1,217 @@
+"""Support for Bosch Alarm Panel binary sensors."""
+
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+from bosch_alarm_mode2 import Panel
+from bosch_alarm_mode2.const import ALARM_PANEL_FAULTS
+
+from homeassistant.components.binary_sensor import (
+    BinarySensorDeviceClass,
+    BinarySensorEntity,
+    BinarySensorEntityDescription,
+)
+from homeassistant.const import EntityCategory
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddConfigEntryEntitiesCallback
+
+from . import BoschAlarmConfigEntry
+from .entity import BoschAlarmAreaEntity, BoschAlarmEntity, BoschAlarmPointEntity
+
+
+@dataclass(kw_only=True, frozen=True)
+class BoschAlarmFaultEntityDescription(BinarySensorEntityDescription):
+    """Describes Bosch Alarm sensor entity."""
+
+    fault: int
+
+
+FAULT_TYPES = [
+    BoschAlarmFaultEntityDescription(
+        key="panel_fault_battery_low",
+        translation_key="panel_fault_battery_low",
```

Comment:
Can we do without the fault prefixes? If so we can just use the device class rtanslation here

Reply:
yep works for me